timbermill-client/src/main/java/com/datorama/oss/timbermill/unit/EventsWrapper.java

```diff
@@ -9,30 +9,31 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME)
 public class EventsWrapper {
```

Comment:
Revert changes

Reply:
I copied the original file from master but it still shows difference in the indentation